src/action.js

```diff
@@ -305,8 +223,114 @@ inherit(Action, Emitter, {
 
 })
 
-Object.defineProperty(Action.prototype, 'type', {
-  get () {
-    return this.command[this.status]
+/**
+ * Complete actions can never change. This is a courtesy warning
+ * @param {Action} action
+ * @param {string} method
+ * @return A function that will warn developers that they can not
+ * update this action
+ */
+function createCompleteWarning (action, method) {
+  return function () {
+    console.assert(false, `Action "${action.command.name || action.type}" is already in ` +
+                   `the ${action.status} state, it will not change. Calling ` +
+                   `${method} will not change it.`)
+  }
```

Comment:
why `console.assert(false, "the string")` vs `console.warn("the string")`? wider browser support or something?